src/components/Book/Book.js

```diff
@@ -1,22 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { connect } from 'react-redux';
 import './book.css';
 
-function Book({ book }) {
+function Book({
+  book, deleteBook,
+}) {
   return (
     <tr>
       <td>{book.id}</td>
       <td>{book.title}</td>
       <td>{book.category}</td>
+      <td><button type="submit" onClick={() => { deleteBook(book.id); }}>Delete Book</button></td>
     </tr>
   );
 }
 
 Book.propTypes = {
-  id: PropTypes.number,
-  title: PropTypes.string,
-  category: PropTypes.string,
-}.isRequired;
+  book: PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    title: PropTypes.string.isRequired,
+    category: PropTypes.string.isRequired,
+  }).isRequired,
+  deleteBook: PropTypes.func.isRequired,
+};
 
-export default Book;
+const mapDispatchToProps = dispatch => ({
```

Comment:
- [ ] This should be moved to `BookList` Component, and pass down `deleteBook` to every `Book`